problem_builder/dashboard.py

```diff
@@ -238,10 +238,26 @@ class DashboardBlock(StudioEditableXBlockMixin, XBlock):
         help=_("Toggles if numeric values are displayed"),
         scope=Scope.content
     )
+    header_html = String(
+        display_name=_("Header HTML"),
+        default="",
+        help=_("Additional HTML to include after the heading."),
```

Comment:
Suggest: "HTML to include before the problem." (or "question" instead of problem - depends on how you refer to this elsewhere)

It's confusing that you call it "header HTML" then say it comes _after_ the heading.